### PR TITLE
rel: 1.0.0

### PR DESCRIPTION
**Pull request by mgxd**
- bump version #s

### CODE REVIEW

**Comment by mgxd** [comment]
@satra @effigies @djarecka testing conda now - if that passes, any final additions before releasing?

File: CHANGES [hunk 1]
```diff
@@ -1,4 +1,7 @@
-Upcoming release (1.0.0)
+Upcoming release
+================
+
```
Comment (effigies) [inline CHANGES]: Maybe let's not add an "Upcoming release" entry until after the release. Just seems weird to have it in the official release docs, even if it'll show up in "latest" on the next commit.

File: CHANGES [hunk 2]
```diff
@@ -1,4 +1,7 @@
...
+1.0.0 (January 24, 2018)
 ========================
```
Comment (effigies) [inline CHANGES]: Link to the milestone? https://github.com/nipy/nipype/milestone/16?closed=1

**Comment by djarecka** [comment]
@mgxd no, I'm not working on anything additional to this right now

**Comment by effigies** [comment]
Just had a look. The [currently running](https://travis-ci.org/conda-forge/healpy-feedstock) OSX conda-forge job was [submitted](https://github.com/conda-forge/healpy-feedstock/pull/8) a week ago.

So that's fun.

File: doc/documentation.rst [hunk 3]
```diff
@@ -9,7 +9,7 @@ Documentation
    :Release: |version|
    :Date: |today|
 
-Previous versions:  `0.13.1 <http://nipype.readthedocs.io/en/0.13.1/>`_ `0.12.1 <http://nipype.readthedocs.io/en/0.12.1/>`_
+Previous versions: `0.14.0 <http://nipype.readthedocs.io/en/v0.14.0/>`_ `0.13.1 <http://nipype.readthedocs.io/en/0.13.1/>`_
```
Comment (satra) [inline doc/documentation.rst]: shouldn't this be 0.14.1

Comment (effigies) [reply doc/documentation.rst]: 0.14.1 became 1.0.

Comment (satra) [reply doc/documentation.rst]: :) i thought we had released 0.14.1 !! now i can understand the slowness of releases.

File: CHANGES [hunk 4]
```diff
@@ -1,6 +1,8 @@
-Upcoming release (1.0.0)
+1.0.0 (January 24, 2018)
```
Comment (effigies) [inline CHANGES]: Should this be 25 now?

Comment (mgxd) [reply CHANGES]: @effigies I've updated it for [the official release](https://github.com/nipy/nipype/releases/tag/v1.0.0)

**Pull request merged**